package.json

```diff
@@ -50,8 +50,9 @@
     "kleur": "^4.1.5",
     "micromatch": "^4.0.5",
     "pirates": "^4.0.5",
-    "playwright-chromium": "=1.37.0",
-    "playwright-core": "=1.37.0",
+    "playwright": "=1.38.0",
```

Comment:
I see that in the end installing playwright in the main package.json has been the way to avoid the rabbit hole related to "cannot find module 'playwright/lib/transformer/esmLoader', I believe is a good choice, the alternative(s) not sure if were worth the effort

Reply:
I don't prefer this approach personally given we are including the PW test runner in the final packed code and having to deal with the bundles ourselves. Would have been a lot easier if matcher is exported.